aws/resource_aws_ses_identity_notification.go

```diff
@@ -0,0 +1,95 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesNotification() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesNotificationSet,
+		Read:   resourceAwsSesNotificationRead,
+		Update: resourceAwsSesNotificationSet,
+		Delete: resourceAwsSesNotificationDelete,
+
+		Schema: map[string]*schema.Schema{
+			"topic_arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: false,
```

Comment:
Also, we need to add a `ValidateFunc: validateArn,` attribute, so that it uses a function to validate the value passed in the configuration. `validateArn` is an existing function that validates the format using a given RegExp, so you don't nee much more here :)

For other attributes, it can be nice to also validate the value passed, using custom functions!